### ISSUE TITLE
[Feature Request] PLEASE FORK SOMEBODY???

### ISSUE THREAD

**Issue by JayDee696969** [a4k-openproject/script.extendedinfo#26]
Can somebody please keep this alive!? This is the best looking TMDB browser still. Would love to see this forked to use TMDBH instead of OpenMeta. Nothing beats the GUI of OpenInfo.

**Comment by drinfernoo** [comment]
There are a million forks. If you want to use one of those, go ahead.

If you want to get TMDbHelper lists in the info dialog, you'll need to use it, and a skin that supports it.

**Comment by JayDee696969** [comment]
Yea, but still cant browse TMDB in the nice GUI like OpenInfo provides. I havent seen any OpenInfo forks that use TMDBH have you??

**Comment by drinfernoo** [comment]
They are two fundamentally different add-ons that work in completely different ways. The question you're asking isn't relevant, as they just don't work similarly at all.

**Comment by JayDee696969** [comment]
I thought OpenMeta was pretty much a TMDB browser with Players Kinda just like TMDBH is? I wish i was better at coding. It just seems like it would be a pretty easy task to switch out the calls in OpenInfo to TMDBH instead of Openmeta. I just figured TMDBH is just more up to date than Openmeta is. And the players for TMDBH are more up to date. And another major improvement is the better Trakt integration in TMDBH. I used Notepad++ to find all the calls to Openmeta in Openinfo. I was able to get a couple buttons working in Openinfo GUI. Ultimately I was hoping to add TMDBH Trakt buttons to Openinfo next to the Play, Add to library buttons. Basically all I wanted was the "all movies" and "All TVshows" GUI/skinned sections in Openinfo to pull up the players from TMDBH instead of OpenMeta, and make the add to library functions add to TMDBH instead, and add anothe button connected to TMDBH Trakt options. I just like browsing with the OpenInfo GUI. Oh well, anyways if you say it cant be done, Ill believe you. Ill stop trying . Thanks anyways.
